app.js

```diff
@@ -0,0 +1,49 @@
+let todos = [
+  {
+    id: 1231,
+    title: "Take out the trash",
+    createdAt: "2020-04-17T23:29:57.350Z",
+    lastUpdated: "2020-04-17T23:29:57.350Z",
+    isComplete: false
+  },
+  {
+    id: 2345,
+    title: "Find a bday present for Tim",
+    createdAt: "2019-04-17T23:29:57.350Z",
+    lastUpdated: "2019-04-17T23:29:57.350Z",
+    isComplete: false
+  }
+];
+
+function createToDo(title){
+let date = new Date();
+let createdAt = date.toISOString();
+let id = Math.floor(Math.random() * 9999);
+let lastUpdated = createdAt;
+const todo = {
+  id,
+  title,
+  createdAt,
+  lastUpdated,
+  isComplete: false
+}
+todos.push(todo)
+for (i = 0; i < todos.length; i++){
```

Comment:
I have no idea what you are trying to do with this for-loop. I don't think this is necessary.

Reply:
I was following along kind of what I was seeing with the video you sent. I saw the for loop there and thought it was necessary. I'm still really struggling with reconciling the logic of js.